src/snippets/icon-snapchat.liquid

```diff
@@ -1 +1,20 @@
-<svg xmlns="http://www.w3.org/2000/svg" width="50" height="50"><path d="M46.775 35.079c-5.805-.957-8.458-6.971-8.544-7.164a1.536 1.536 0 0 0-.056-.132c-.175-.354-.352-.851-.204-1.2.256-.604 1.462-.987 2.185-1.216.254-.081.495-.158.684-.232 1.752-.692 2.629-1.601 2.609-2.703-.017-.887-.696-1.697-1.694-2.05a3 3 0 0 0-1.148-.223c-.275 0-.688.039-1.087.225-.667.313-1.256.481-1.672.5a1.973 1.973 0 0 1-.234-.024l.042-.687c.196-3.112.44-6.984-.611-9.342-3.098-6.942-9.669-7.481-11.613-7.481l-.883.008c-1.936 0-8.493.539-11.587 7.477-1.052 2.358-.808 6.229-.612 9.339l.008.12.035.566c-.433.078-1.28-.066-2.158-.477-1.195-.557-3.349.18-3.646 1.741-.131.692.029 2.003 2.575 3.01.19.076.431.153.687.233.72.229 1.925.611 2.182 1.216.148.35-.028.845-.236 1.271-.109.255-2.746 6.266-8.564 7.224-.742.12-1.27.777-1.228 1.534.012.201.06.401.143.596.528 1.236 2.445 2.087 6.025 2.672.062.208.131.521.169.696.077.352.157.714.267 1.089.104.354.468 1.178 1.603 1.178.342 0 .719-.074 1.119-.152.596-.117 1.338-.262 2.291-.262.53 0 1.078.046 1.628.138 1.016.169 1.935.818 3.001 1.571 1.664 1.177 3.55 2.51 6.475 2.51.077 0 .155-.002.23-.007.108.005.216.007.326.007 2.924 0 4.812-1.334 6.479-2.512 1.017-.72 1.978-1.399 2.995-1.569a10.067 10.067 0 0 1 1.63-.138c.919 0 1.644.118 2.292.245.464.091.83.135 1.169.135.759 0 1.339-.435 1.55-1.159.11-.37.19-.724.268-1.082.029-.135.103-.47.169-.692 3.58-.584 5.498-1.434 6.022-2.661.084-.194.133-.397.146-.608a1.467 1.467 0 0 0-1.227-1.528z"/></svg>
\ No newline at end of file
+<svg xmlns="http://www.w3.org/2000/svg" width="20" height="20" viewBox="0 0 512.016 512.016" style="enable-background:new 0 0 512.016 512.016;" class="icon">
```

Comment:
Can you remove the added `style` attribute and remove the multiple lines?

Reply:
@huguestennier Fixed